### PR TITLE
feat(48165): remove semicolon from skills

### PR DESCRIPTION
**Pull request by gabrielgeorge**
Remove semicolon from skills and also change it to `|`

### CODE REVIEW

File: src/templates/person.js [hunk 1]
```diff
@@ -113,11 +113,17 @@ const Person = ({ data }) => {
             </dl>
             <dl>
               <dt>Skills:</dt>
-              {advancedSkills.map((skill, i) => (
-                <dd key={`a${i}`}> {skill};</dd>
+              {advancedSkills.map((skill, i, arr) => (
+                <dd key={`a${i}`}>
```
Comment (JeanThirion) [inline src/templates/person.js]: Can you make the key name a bit more explicit ? (same for intermediate) 
I know you didn't do it in the first place but i missed it

File: src/templates/person.js [hunk 2]
```diff
@@ -113,11 +113,17 @@ const Person = ({ data }) => {
...
+                  {skill}
+                  {arr.length - 1 !== i && <span className="skill-separator"> | </span>}
```
Comment (JeanThirion) [inline src/templates/person.js]: I'm being picky but could you please invert the boolean expression ? Much more readable the other way around 
(i !== arr.length - 1)

Comment (gabrielgeorge) [reply src/templates/person.js]: done :)

File: src/templates/person.js (hunk 1 above)
Comment (gabrielgeorge) [reply src/templates/person.js]: done 👍

**Pull request merged**